src/can2040.c

```diff
@@ -3,6 +3,7 @@
 // Copyright (C) 2022-2025  Kevin O'Connor <kevin@koconnor.net>
 //
 // This file may be distributed under the terms of the GNU GPLv3 license.
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Am I correct to assume it's GPL-3.0-only, and not -and-later?

Reply:
Zephyr build system West uses SPDX IDs to keep track of license of each file compiled, so this is important to make sure whoever uses this driver in Zephyr (or if upstreaming fails as my out-of-tree glue code) can get your licensing information and stay compliant with it.